pandas/tests/dtypes/test_common.py

```diff
@@ -322,9 +325,13 @@ def test_is_datetimelike():
     assert com.is_datetimelike(s)
 
 
+integer_dtypes = []  # type: List[Union[Series, str, str, Any, ExtensionDtype]
```

Comment:
You should only need `str` once within the `Union`. `Any` is also a little too loose - can that not be made more specific?

Reply:
Also I think you can just add all of the elements this should contain here, rather than defining and adding this later